project2/Vehicle/project2Uno.ino

```diff
@@ -0,0 +1,36 @@
+
+String commandFormat(String comm) {
+    return "A<" + comm + ">";
+}
+
+void setup() {
+    Serial.begin(9600);
+}
+
+void loop() {
+    char comm;
+    if (Serial.available()) {
+        comm = Serial.read();
+        if(comm != '\0') {
+        Serial.println(comm);
```

Comment:
Shouldn't this be printing `A F` where `F` is the command sent? Also should be indented.
```suggestion
            Serial.println(comm);
```